source/docs/software/wpilib-tools/pathweaver/integrating-robot-program.rst

```diff
@@ -5,15 +5,29 @@ The ``TrajectoryUtil`` class can be used to import a PathWeaver JSON into robot
 
 The ``fromPathweaverJson`` (Java) / ``FromPathweaverJson`` (C++) static methods in ``TrajectoryUtil`` can be used to create a trajectory from a JSON file stored on the roboRIO file system.
 
-.. note:: JSON files placed in ``src/main/deploy`` will automatically be placed on the roboRIO file system in ``/home/lvuser/deploy``.
+.. note:: PathWeaver places JSON files in ``src/main/deploy/paths`` which will automatically be placed on the roboRIO file system in ``/home/lvuser/deploy/paths`` and can be accessed using getDeployDirectory as shown below.
 
 .. tabs::
 
    .. code-tab:: java
 
-      Trajectory trajectory = TrajectoryUtil.fromPathweaverJson(Paths.get("/home/lvuser/deploy/YourPath.wpilib.json"));
+      String trajectoryJSON = "paths/YourPath.wpilib.json";
+      try {
+        Path trajectoryPath = Filesystem.getDeployDirectory().getPath().resolve(trajectoryJSON);
+        Trajectory trajectory = TrajectoryUtil.fromPathweaverJson(trajectoryPath);
+      } catch (IOException ex) {
+        DriverStation.reportError("Unable to open trajectory: " + trajectoryJSON, ex.getStackTrace());
+      }
 
    .. code-tab:: c++
 
-      frc::Trajectory trajectory = frc::TrajectoryUtil::FromPathweaverJson("/home/lvuser/deploy/YourPath.wpilib.json");
+       #include <frc/Filesystem.h>
+       #include <frc/trajectory/TrajectoryUtil.h>
+       #include <wpi/Path.h>
 
+       wpi::SmallVector<char, 64> deployDirectory;
```

Comment:
This can be changed to `wpi::SmallString<64>` and add a `wpi/SmallString.h` include.